app/services/migration/migrators/user.rb

```diff
@@ -33,16 +33,13 @@ def call
         trns = unique_validated_trns(ecf_user)
         validate_multiple_trns!(trns, user)
 
-        emails = unique_emails(ecf_user)
-        validate_multiple_emails!(emails, user)
-        validate_existing_email_match!(emails.first, user)
-
         npq_application = most_recent_updated_npq_application(ecf_user)
```

Comment:
I wonder if this should be by created at rather than updated at, as we might have an older application updated today, but a newer one created which has the latest data 🤔 what do you think?

Reply:
Ok, i've updated to `created_at`

Reply:
thanks could we update the method name as well :)

Reply:
Updated